src/main/scala/temple/builder/project/ProjectBuilder.scala

```diff
@@ -28,6 +29,13 @@ object ProjectBuilder {
         }
     }
 
-    Project(databaseCreationScripts)
+    val dockerfiles = templefile.services.zip(80 until Int.MaxValue).map {
```

Comment:
I'm envisaging the port will eventually come from elsewhere, since it'll also be needed for code generation. For now this seemed the nicest way to express it.

Reply:
I suppose this works for now - but definitely in future we probably want a more sophisticated scheme than just incrementing the port - lots of ports we don't want to get confused about as they have conventionally different uses

Reply:
Can we start at a number greater than 1024? That way we won’t run into any conflicts with expected usage. E.g. browsers may try to connect to 443 from 80

Reply:
Sure - but remember this is a port used in a Docker container. How that's exposed to the host machine is another configuration entirely.